include/event_manager.h

```diff
@@ -273,8 +276,8 @@ extern const struct event_type __stop_event_types[];
  * @param print_fn  		Function to stringify event of this type.
  * @param ev_info_struct	Data structure describing event type.
  */
-#define EVENT_TYPE_DEFINE(ename, log_fn, ev_info_struct) \
-	_EVENT_TYPE_DEFINE(ename, log_fn, ev_info_struct)
+#define EVENT_TYPE_DEFINE(ename, init_log_en, log_fn, ev_info_struct) \
```

Comment:
Please update documentation of `EVENT_TYPE_DEFINE` macro. 
Describe new parameter, also update `print_fn` to `log_fn`.

Reply:
done